adapter/redisquota/redisquota_test.go

```diff
@@ -478,7 +481,9 @@ func TestBadConfig(t *testing.T) {
 
 func TestErrorResponse(t *testing.T) {
 	mockredis := testServer{errOnStart: false, errOnIntRet: true}
+	mockredis.Mutex.Lock()
```

Comment:
There is only one `errOnIntRet` server for now, in order to cover the error handling code (basically to improve the code coverage). But the coverage is still not satisfying... May need better solution.
My understanding is that it is enforcing that there is only one server started at this time with localhost:9736 (correct me if I'm wrong). ":0" seems do not accepted by the redeo server setting. But actually I can remove the port number here, and it will use `0.0.0.0:9736` as default config. 
I'm also wondering if grafana is using any specific port?